systers_portal/meetup/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from meetup.views import MeetupLocationDetailView
+
+
+urlpatterns = [
+    url(r'^(?P<slug>\w+)/(?P<page_slug>\w+)/$',
```

Comment:
So the Django way: it is a bad practice to construct the views directly in the url config (as you said), because suddenly the url config has a view (similar snippet is somewhere in the official docs, but definitely a bad practice).

If the view is slim, that's perfect! With time it might grow actually, that's why let's keep it in `views.py` :)